docs/_documentation/configuration.md

```diff
@@ -21,6 +21,14 @@ basic usage.
 }
 ~~~~
 
+## Portable Mode {#portableMode}
+
+If you want your configuration within your application folder for portability just
+create a `portable` directory inside of it. You can copy your configuration files over
+from your home directory if they already exist.
+
+Portable mode overrides ZAZU_HOME, if defined.
```

Comment:
`ZAZU_HOME` isn't documented and was intended for testing purposes. I don't think we should mention it.